unblob/processing.py

```diff
@@ -143,6 +143,17 @@ def __init__(
         self.size = size
         self.result = result
 
+        self.extract_dir = self._get_extract_dir()
+
+        assert self.extract_dir.is_absolute()
```

Comment:
no assert here either, check it and handle problem if exists

Reply:
it is for documentation only - below we can already verify what the `extract_dir` is, but I prefer to treat even helper functions as black boxes

Reply:
removed